menas/src/main/scala/za/co/absa/enceladus/menas/controllers/SchemaController.scala

```diff
@@ -163,6 +181,15 @@ class SchemaController @Autowired()(
         throw notFound()
     }
   }
+
+  @GetMapping(path = Array("/availability"))
+  @ResponseStatus(HttpStatus.OK)
+  def getAvailability(): CompletableFuture[SchemaApiAvailability] = {
+    val registryAvailable = schemaRegistryService.schemaRegistryBaseUrl.isDefined
+
+    // the ability to upload or input a remoteUrl is always enabled, so it is not part of the availability endpoint at the moment
+    Future { SchemaApiAvailability(registry = registryAvailable)}
```

Comment:
This does not need to be executed in a separate thread since the value is already known.
```suggestion
    Future.successful(SchemaApiAvailability(registry = registryAvailable))
```

Reply:
True, changed.